esctest/tests/decset.py

```diff
@@ -436,7 +444,7 @@ def test_DECSET_OPT_ALTBUF_CURSOR(self):
     self.doAltBuftest(esccmd.OPT_ALTBUF_CURSOR, True, True, True)
 
   # xterm doesn't implement auto-wrap mode when wide characters are disabled.
```

Comment:
Is this comment still true? If not, remove it; if so, why remove `@optionRejects`?

Reply:
I overlooked the comment :-)